Error while building with Maven

Since eea6e879694656aa575c08665b36301a8df88d10, I can't build it anymore. GitLab builds are also failing.

```
Jarnes-iMac:Nemisys Jarne$ mvn clean && mvn package
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building nemisys 1.0dev
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ nemisys ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.209 s
[INFO] Finished at: 2016-10-05T14:35:44+02:00
[INFO] Final Memory: 6M/123M
[INFO] ------------------------------------------------------------------------
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building nemisys 1.0dev
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:1.4.1:enforce (enforce) @ nemisys ---
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ nemisys ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 4 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ nemisys ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 176 source files to /Users/Jarne/Downloads/Nemisys/target/classes
[INFO] -------------------------------------------------------------
[WARNING] COMPILATION WARNING : 
[INFO] -------------------------------------------------------------
[WARNING] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/utils/ConfigSection.java: Some input files use unchecked or unsafe operations.
[WARNING] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/utils/ConfigSection.java: Recompile with -Xlint:unchecked for details.
[INFO] 2 warnings 
[INFO] -------------------------------------------------------------
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/SessionManager.java:[3,23] package io.netty.buffer does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/SessionManager.java:[4,31] package io.netty.channel.socket does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[3,26] package io.netty.bootstrap does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[4,23] package io.netty.buffer does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[5,24] package io.netty.channel does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[6,24] package io.netty.channel does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[7,24] package io.netty.channel does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[8,24] package io.netty.channel does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[9,28] package io.netty.channel.nio does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[10,35] package io.netty.channel.socket.nio does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[27,38] cannot find symbol
  symbol: class ChannelInboundHandlerAdapter
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[30,15] cannot find symbol
  symbol:   class Bootstrap
  location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[31,15] cannot find symbol
  symbol:   class EventLoopGroup
  location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[32,15] cannot find symbol
  symbol:   class Channel
  location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[34,60] package io.netty.channel.socket does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[70,35] package io.netty.channel.socket does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[84,29] cannot find symbol
  symbol:   class ChannelHandlerContext
  location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[89,33] cannot find symbol
  symbol:   class ChannelHandlerContext
  location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/SessionManager.java:[144,9] cannot find symbol
  symbol:   class DatagramPacket
  location: class org.itxtech.nemisys.raknet.server.SessionManager
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/SessionManager.java:[146,13] cannot find symbol
  symbol:   class ByteBuf
  location: class org.itxtech.nemisys.raknet.server.SessionManager
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[47,29] cannot find symbol
  symbol:   class Bootstrap
  location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[48,25] cannot find symbol
  symbol:   class NioEventLoopGroup
  location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[51,30] cannot find symbol
  symbol:   class NioDatagramChannel
  location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[79,58] package io.netty.channel.socket does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[79,74] cannot find symbol
  symbol:   variable Unpooled
  location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[83,5] method does not override or implement a method from a supertype
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[85,50] package io.netty.channel.socket does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[88,5] method does not override or implement a method from a supertype
[INFO] 28 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.913 s
[INFO] Finished at: 2016-10-05T14:35:47+02:00
[INFO] Final Memory: 16M/201M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project nemisys: Compilation failure: Compilation failure:
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/SessionManager.java:[3,23] package io.netty.buffer does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/SessionManager.java:[4,31] package io.netty.channel.socket does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[3,26] package io.netty.bootstrap does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[4,23] package io.netty.buffer does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[5,24] package io.netty.channel does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[6,24] package io.netty.channel does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[7,24] package io.netty.channel does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[8,24] package io.netty.channel does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[9,28] package io.netty.channel.nio does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[10,35] package io.netty.channel.socket.nio does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[27,38] cannot find symbol
[ERROR] symbol: class ChannelInboundHandlerAdapter
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[30,15] cannot find symbol
[ERROR] symbol:   class Bootstrap
[ERROR] location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[31,15] cannot find symbol
[ERROR] symbol:   class EventLoopGroup
[ERROR] location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[32,15] cannot find symbol
[ERROR] symbol:   class Channel
[ERROR] location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[34,60] package io.netty.channel.socket does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[70,35] package io.netty.channel.socket does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[84,29] cannot find symbol
[ERROR] symbol:   class ChannelHandlerContext
[ERROR] location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[89,33] cannot find symbol
[ERROR] symbol:   class ChannelHandlerContext
[ERROR] location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/SessionManager.java:[144,9] cannot find symbol
[ERROR] symbol:   class DatagramPacket
[ERROR] location: class org.itxtech.nemisys.raknet.server.SessionManager
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/SessionManager.java:[146,13] cannot find symbol
[ERROR] symbol:   class ByteBuf
[ERROR] location: class org.itxtech.nemisys.raknet.server.SessionManager
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[47,29] cannot find symbol
[ERROR] symbol:   class Bootstrap
[ERROR] location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[48,25] cannot find symbol
[ERROR] symbol:   class NioEventLoopGroup
[ERROR] location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[51,30] cannot find symbol
[ERROR] symbol:   class NioDatagramChannel
[ERROR] location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[79,58] package io.netty.channel.socket does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[79,74] cannot find symbol
[ERROR] symbol:   variable Unpooled
[ERROR] location: class org.itxtech.nemisys.raknet.server.UDPServerSocket
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[83,5] method does not override or implement a method from a supertype
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[85,50] package io.netty.channel.socket does not exist
[ERROR] /Users/Jarne/Downloads/Nemisys/src/main/java/org/itxtech/nemisys/raknet/server/UDPServerSocket.java:[88,5] method does not override or implement a method from a supertype
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

--- a/pom.xml
+++ b/pom.xml
@@ -42,7 +42,7 @@
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-all</artifactId>
-            <version>4.1.4.Final</version>
+            <version>4.1.5.Final</version>
         </dependency>
     </dependencies>
 

--- a/pom.xml
+++ b/pom.xml
@@ -39,6 +39,11 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-all</artifactId>
+            <version>4.1.4.Final</version>
+        </dependency>
     </dependencies>
 
     <build>

